app/models/podcast.rb

```diff
@@ -5,7 +5,7 @@ class Podcast < ApplicationRecord
   has_many :episodes, dependent: :destroy
 
   def path
-    '/podcast'
+    "/podcasts/#{slug}"
```

Comment:
Is `slug` a required value?  What happens if slug is blank?

Reply:
That's a good point / question. 👍🏻

I think it's not currently required by code, but has been by social practice. We should codify it though. Because weird shit happens without one.

We can do that in a separate PR.
All current podcasts have slugs and new podcasts don't come around that often.